src/app/technical-record/body/body.component.html

```diff
@@ -17,6 +17,57 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType == 'psv'">
```

Comment:
update all to use the `===`

Reply:
let's break body into children components that render the fields needed per vehicle types like it is done brakes.component (that is what we want)

We want the following (as folder structure):
```
/body
  body.component.html // will load body-psv fields in here
   ...
  /body-psv
    body-psv.component.html //only contains the body fields for psv 
```

```
// body-psv.component should look like what we have in here
// if we refer to the all structure mapping it has model literal, chassis make, chassis model fields
<div>
   <dt class="govuk-summary-list__key">
      Make
    </dt>
    <dd id="test-bodyMake" class="govuk-summary-list__value">
      {{ activeRecord.bodyMake | DefaultNullOrEmpty }}
    </dd>
    <dd class="govuk-summary-list__value"></dd>
     ..... model literal fields, etc...
</div>
```

Then body.omponent.html loads the common body fields + fields conditionally to specific vehicle types in our case psv:

```
// body.component.html
<div>
  ...field for model here
  ...field for make
  ...body type field
  ...function field
  ...conversion reference number 
  <ng-container *ngIf="currentVehicleType === 'psv'"> // this only contains fields for psv (model literal, chassis make, chassis model)
     <vtm-body-psv  [body]="body" />
  </ng-container>
</div>

```